pages/quiz-result.html

```diff
@@ -4,9 +4,67 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://maple-ui.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="../CSS/quiz-category.css">
+    <link rel="stylesheet" href="../CSS/util.css">
+    <title>Result</title>
```

Comment:
Is it possible to import all the CSS files to one main file? If so do that.

Reply:
Thank you for pointing out planning to do in future but update now only😃